poc/field40.py

```diff
@@ -0,0 +1,26 @@
+from typing import Self
+
+from sage.all import GF
+from vdaf_poc.field import NttField
+
+
+class Field40(NttField):
```

Comment:
nit (no strong opinion): perhaps we put it in the same file as `Field32`, just name it `field.py`, or `pine_field.py`, and have both Field32 and Field40 there.

Reply:
Agreed.